theseus/labs/lie/functional/se3_impl.py

```diff
@@ -112,25 +102,35 @@ def randn(
 ) -> torch.Tensor:
     if len(size) != 1:
         raise ValueError("The size should be 1D.")
-    rotation = so3.randn(
-        size[0],
-        generator=generator,
-        dtype=dtype,
-        device=device,
-    )
+    rotation = so3.randn(size[0], generator=generator, dtype=dtype, device=device)
     translation = torch.randn(
-        size[0],
-        3,
-        1,
-        generator=generator,
-        dtype=dtype,
-        device=device,
+        size[0], 3, 1, generator=generator, dtype=dtype, device=device
     )
     ret = torch.cat((rotation, translation), dim=2)
     ret.requires_grad_(requires_grad)
     return ret
 
 
+# -----------------------------------------------------------------------------
+# Identity
+# -----------------------------------------------------------------------------
+def identity(
+    *size: int,
+    dtype: Optional[torch.dtype] = None,
+    device: constants.DeviceType = None,
+    requires_grad: bool = False,
+) -> torch.Tensor:
+    if len(size) != 1:
+        raise ValueError("The size should be 1D.")
+    ret = torch.tensor(
+        [[1.0, 0.0, 0.0, 0.0], [0.0, 1.0, 0.0, 0.0], [0.0, 0.0, 1.0, 0.0]],
```

Comment:
Call into SO3.identity or keep it independent?

Reply:
I don't think we have `SO3.eye`, we can alias although I have not seen this commonly used.

Reply:
This was actually proposed by @fantaosha and seems like a more straightforward solution to me (it requires one less function call).

I actually thought about naming it `SO3.eye` as well, but I wasn't sure. I do like it though, it's very concise.

Reply:
Based on offline discussion we will only keep identity for now and add eye if user feedback surfaces the need.